newIDE/app/src/InstancesEditor/index.js

```diff
@@ -457,6 +457,16 @@ export default class InstancesEditor extends Component<Props> {
     this.setZoomFactor(this.getZoomFactor() + value);
   }
 
+  zoomOnMouseBy(value: number) {
+    const beforeZoomCursor = this.getLastCursorSceneCoordinates();
+    this.setZoomFactor(this.getZoomFactor() + value);
+    const afterZoomCursor = this.getLastCursorSceneCoordinates();
+    this.viewPosition.scrollBy(beforeZoomCursor[0] - afterZoomCursor[0], beforeZoomCursor[1] - afterZoomCursor[1]);
```

Comment:
Let's add a quick comment `// Compensate for the cursor change in position, so that the zoom is done "centered" on the cursor`.